iina/MainWindowController.swift

```diff
@@ -49,6 +49,8 @@ fileprivate extension NSStackView.VisibilityPriority {
   static let detachEarliest = NSStackView.VisibilityPriority(rawValue: 750)
 }
 
+// The minimum distance that the user must drag before their click or tap gesture is interpreted as a drag gesture:
+fileprivate let minimumInitialDragDistance: CGFloat = 4.0
```

Comment:
Please add some comments to clarify that the intended distance is 2 instead of 4 since we are using spared distance.